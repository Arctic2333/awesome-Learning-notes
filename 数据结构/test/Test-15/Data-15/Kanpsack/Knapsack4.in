5  10.5
物品1  9.2  7.0
物品2  2.0  2.0
物品3  4.0  2.2
物品4  1.0  1.0
物品5  10.0  4.5
