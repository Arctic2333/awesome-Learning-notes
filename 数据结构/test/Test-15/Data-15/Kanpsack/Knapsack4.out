物品5: 4.5Kg
物品3: 2.2Kg
物品1: 3.8Kg
Total price: 18.99
